t/lib/server.lua

```diff
@@ -85,6 +85,15 @@ function _M.opentracing()
 end
 
 
+function _M.with_header()
+    ngx.header['Content-Type'] = 'application/xml'
+    ngx.header['X-Server-id'] = 100
+    --split into multiple chunk
```

Comment:
I made a test right now, unable to generate multiple `chunk` responses.

We need another way.